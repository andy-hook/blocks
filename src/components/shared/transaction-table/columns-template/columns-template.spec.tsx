import React from "react"
import ColumnsTemplate from "./columns-template"
import { renderWithTheme } from "@test-utils"

jest.mock("react-truncate-string")

describe("<ColumnsTemplate />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      "dark",
      <ColumnsTemplate
        block="string"
        hash="string"
        fromTo="string"
        value="string"
      />
    )
    expect(tree).toMatchSnapshot()
  })
})
