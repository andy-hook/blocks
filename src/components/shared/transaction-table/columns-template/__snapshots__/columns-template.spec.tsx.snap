// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ColumnsTemplate /> renders correctly 1`] = `
Array [
  <td
    className="sc-bdVaJa sc-bwzfXH dfeDEW"
  >
    string
  </td>,
  <td
    className="sc-bdVaJa sc-htpNat jDGsFT"
  >
    string
  </td>,
  <td
    className="sc-bdVaJa sc-bxivhb hxXuRT"
  >
    string
  </td>,
  <td
    className="sc-bdVaJa sc-ifAKCX gEVBXo"
  >
    string
  </td>,
]
`;
